### PR TITLE
add dismountAction if unit is in vehicule before surrenderAction

### PR DESCRIPTION
**Pull request by zalexki**

### CODE REVIEW

**Comment by Sparker95** [comment]
That's nice. Did you test it? Do they really dismount vehicles when you give them this goal?

File: Project_0.Altis/Misc/fn_actionDropAllWeaponsAndSurrender.sqf [hunk 1]
```diff
@@ -11,7 +11,9 @@ Author: Sen 01.03.2019
 */
 params ["_unit"];
 
-private _currentWeapon = currentWeapon _unit;
+_unit setCombatMode "BLUE";
```
Comment (Sparker95) [inline Project_0.Altis/Misc/fn_actionDropAllWeaponsAndSurrender.sqf]: I am pretty sure that setCombatMode is a group setting.
https://community.bistudio.com/wiki/setCombatMode

**Comment by Jeroen-Notenbomer** [comment]
Removed the ai setBehavior lines as they where probably not needed.

File: Project_0.Altis/Misc/fn_actionDropAllWeaponsAndSurrender.sqf (hunk 1 above)
Comment (Jeroen-Notenbomer) [reply Project_0.Altis/Misc/fn_actionDropAllWeaponsAndSurrender.sqf]: Often when a group is requested for a function unit can be used as well(unit cant exist without group).

**Pull request merged**